core/src/apps/nostr/__init__.py

```diff
@@ -0,0 +1,10 @@
+if not __debug__:
+    from trezor import utils
+
+    utils.halt("Disabled in production mode")
+
+from apps.common.paths import PATTERN_BIP44
+
+CURVE = "secp256k1"
+SLIP44_ID = 1237
+PATTERN = PATTERN_BIP44
```

Comment:
why would Nostr use a BIP44 5-part pattern?

is there any reason for unhardened derivations at all? putting multiple pubkeys under the same "account", which still need to be "discoverable" together?

Reply:
update: we opened https://github.com/nostr-protocol/nips/issues/1774 and we could write another NIP to specify a better path, but there is no point pausing this PR any longer than it has already been paused.

please add a comment here that this path is provisional and we don't want it, then let's merge

Reply:
Added comment here and at the very beginning of the PR description.